ixwebsocket/IXSelectInterruptFactory.cpp

```diff
@@ -20,7 +20,7 @@ namespace ix
 #if defined(__linux__) || defined(__APPLE__)
         return ix::make_unique<SelectInterruptPipe>();
 #else
```

Comment:
same here.

Reply:
The SelectInterruptEvent code works for all platforms because it has a fall back "timeout based" branch if _WIN32 isn't defined. That's what the "(see...)" comment was for that I stopped writing in the middle of the comment.